arch/arm/dts/tegra186-p2771-0000.dtsi

```diff
@@ -12,6 +12,14 @@
 		sdhci0 = "/sdhci@3460000";
 		sdhci1 = "/sdhci@3400000";
 		i2c0 = "/bpmp-i2c";
+		i2c1 = "/i2c@3160000";
```

Comment:
I'd suggest reviewing the schematic to determine which buses are useful; there may be useful buses that don't have anything attached by default, and there could be buses with devices attached that likely aren't useful.